### PR TITLE
Support toggle segmentation mask

### PR DESCRIPTION
**Pull request by oeway**
This PR allows switch on/off the segmentation mask, either click the checkbox or use space.

### CODE REVIEW

File: src/SingleCellAnnotator.imjoy.html [hunk 1]
```diff
@@ -941,7 +949,7 @@
               color: 'rgba(255, 255, 255, 0)'
             }),
             stroke: new Stroke({
-                color: this.current_task_name ==='segmentation'? '#f5d80b91': '#448aff53',
+                color: this.show_segmentation? '#00000000' : (this.current_task_name ==='segmentation'? '#f5d80b91': '#448aff53'),
```
Comment (haoxusci) [inline src/SingleCellAnnotator.imjoy.html]: change to below?
color: this.current_task_name ==='segmentation'? (this.show_segmentation? '#f5d80b91': '#00000000'): '#448aff53' ,

Comment (haoxusci) [reply src/SingleCellAnnotator.imjoy.html]: suggested changed works in my test

Comment (oeway) [reply src/SingleCellAnnotator.imjoy.html]: I have it fixed in another PR

Comment (oeway) [reply src/SingleCellAnnotator.imjoy.html]:
```suggestion
                color: this.current_task_name ==='segmentation'? (this.show_segmentation? '#f5d80b91': '#00000000'): '#448aff53',
```

**Pull request merged**